### DISCUSSION TITLE
Migration guide to v0.8.0

### DISCUSSION THREAD

**Discussion by jfmou** [GoogleChrome/lighthouse-ci#630] [Q&A] [answered]
I'm not sure i've looked everywhere but I can't find the migration guide to update lhci server from 0.7.0 to 0.8.0.

Is there any customisation needed / things to know ?

Thank you !

**Comment by patrickhulce** [comment, marked as answer]
There isn't a migration guide yet, but the breaking changes are captured in the release notes.

[LHCI release notes](https://github.com/GoogleChrome/lighthouse-ci/releases/tag/v0.8.0)
[LH release notes](https://github.com/GoogleChrome/lighthouse/releases/tag/v8.0.0)

The migration guide will likely be identical to the previous versions.

Affected Usage Patterns
lighthouse:* assertion preset usage (new audits asserted, assertions could fail)
assert usage (some audits removed and scores changed, assertions could fail)
If your use of Lighthouse CI doesn't follow any of these patterns, you will likely see no breaking changes.

Breaking Changes
Lighthouse 8.0.0 is now used, several category score changes and different audits in the recommended preset.
